src/Catty/DataModel/Configuration/PenConfiguration.swift

```diff
@@ -22,7 +22,22 @@
 
 struct PenConfiguration {
     var penDown = false
-    var size = SpriteKitDefines.defaultPenSize
+
+    private var _size = SpriteKitDefines.defaultPenSize
+    var size: CGFloat {
+        set {
+            _size = 0.634 * newValue
```

Comment:
Please add this conversion factor to a static let property (e.g., `sizeConversionFactor`) which can also be reused in the tests. Also please adapt the `defaultPenSize` to this conversion factor so that you do not need the separate `_size` property.